### ISSUE TITLE
Structured outputs (lineups & statistics)

### ISSUE THREAD

**Issue by mcarroll316** [DimaKudosh/pydfs-lineup-optimizer#394]
Do the following have structured output options?
1. for lineup in optimizer.optimize(10):
    print(lineup)
2. print_statistic()

Each of those result in a text blob that I've built a parser for (not great). I'm not much of a python guy so trying to avoid modifications to the pydfs code.

**Comment by chanzer0** [comment]
`optimizer.export('result.csv')`

**Comment by mcarroll316** [comment]
No. That gets me the lineups in the format to be uploaded directly into DK, for example. What I was looking for is have a structured output that includes the team, salary, player points, etc., just as you'd see when you print the lineups. Same goes for the statistics (players used) output. I built a custom parser and all is well now. Thanks.

**Comment by mcarroll316** [comment]
I should've mentioned that I've implemented this in my site with a fully-functional UI for the optimizer, which is why I needed to pretty-up the print outputs.

moonshotsmlb dot com

**Comment by Denwen12** [comment]
That is actually pretty easy since I already did that for ownership, I can
build you a exporter for that in the morning
